stdlib/arg.ml

```diff
@@ -309,8 +309,13 @@ let second_word s =
     else if s.[n] = ' ' then loop (n+1)
     else n
   in
-  try loop (String.index s ' ')
-  with Not_found -> len
+  match String.index s '\t' with
+  | n -> loop (n+1)
+  | exception Not_found ->
+      begin match String.index s ' ' with
+      | n -> loop n
```

Comment:
the use of both `loop (n + 1)` and `loop n` confused me a little, maybe you could use `loop (n + 1)` in both cases?